setup.py

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env python
+
+
+from setuptools import setup
+
+setup()
```

Comment:
I don't believe this is required with recent versions of `setuptools` (although I don't know how recent that change is, nor do we have a formal stance on what versions we're committing to supporting)